axelrod/tests/strategies/test_player.py

```diff
@@ -124,6 +121,20 @@ def test_noisy_play(self):
         self.assertEqual(player1.history[0], D)
         self.assertEqual(player2.history[0], D)
 
+    def test_update_history(self):
```

Comment:
Why do we need this? (Not suggesting we don't, just curious what we were/are missing.)